oe_theme.theme

```diff
@@ -932,6 +932,33 @@ function oe_theme_preprocess_pattern_file_translation__preview(array &$variables
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_list_item_block_one_column(array &$variables): void {
+  foreach ((array) $variables['items'] as $key => $value) {
+    $variables['items'][$key]['#fields']['extra_classes'] = 'ecl-col-12';
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_list_item_block_two_columns(array &$variables): void {
+  foreach ((array) $variables['items'] as $key => $value) {
+    $variables['items'][$key]['#fields']['extra_classes'] = 'ecl-col-6';
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_list_item_block_three_columns(array &$variables): void {
+  foreach ((array) $variables['items'] as $key => $value) {
+    $variables['items'][$key]['#fields']['extra_classes'] = 'ecl-col-4';
```

Comment:
`extra_classes` is not defined as a pattern field so it will be ignored. Let's not do this via but directly in the Twig template.

Reply:
I changed 3 times to way to handle that. Now it's working.